website/client/js/services/achievementServices.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+/**
+ * Services that handle achievement logic.
+ */
+
+angular.module('habitrpg').factory('Achievement',
+['$rootScope', function($rootScope) {
+
+  function displayAchievement(achievementName) {
+    $rootScope.openModal('achievements/' + achievementName,
+      {controller:'UserCtrl', size:'sm'});
+  }
+
+  function displayBulkyAchievement(achievementName) {
+    $rootScope.openModal('achievements/' + achievementName,
+      {controller:'UserCtrl'});
```

Comment:
I would like some input on whether or not this is the best way to handle passing vs. not passing the `size` value.  I wasn't sure if it was desirable for the caller to specify the modal size, or if it would be better to abstract that away.  Since the goal of this whole PR  is to abstract stuff away, I went with the less transparent option.

However, I think it would be preferable if one of these methods called the other under the hood with somehow modified parameters.  This would make unit testing cleaner and reduce duplicate functionality.